vita3k/lang/include/lang/state.h

```diff
@@ -331,10 +331,10 @@ struct LangState {
         };
         std::map<std::string, std::string> license_install = {
             { "select_license_type", "Select license type" },
-            { "select_bin_rif", "Select work.bin/rif" },
+            { "select_bin_rif", "Select work.bin" },
```

Comment:
revert

Reply:
What? In the image you sent, it's really close to the border of the button. It would be so much better if "/rif" was not there.

Reply:
What????
a. It looks super unclean, it looks like the "f" if "rif" has been cropped already
b. work.bin is the same as a "rif file".

I refuse to change this.

Reply:
> What???? a. It looks super unclean, it looks like the "f" if "rif" has been cropped already b. work.bin is the same as a "rif file".
> 
> I refuse to change this.

u no understand by my mean, did you see `license_install`?

Reply:
i do not intend on fixing any other issues in this pr. this pr has done what i wanted it to do.

`license_install` is not in the state.h file, make your own pr and add it, i dont know how it works properly.

Again, I refuse to change it.

Reply:
Mr.@Zangetsu38 please check this pr

Reply:
which file? there 40 things here

Reply:
> What???? a. It looks super unclean, it looks like the "f" if "rif" has been cropped already b. work.bin is the same as a "rif file".
> 
> I refuse to change this.

Revert, rif exist, is licence file